### PR TITLE
Transfer

### PR DESCRIPTION
**Pull request by Yoel-Perman-Brilliant**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+  @Override
+  public void robotInit() {
+    robotNamespace = new RootNamespace("robot");
+
+    transfer = new MotoredGenericSubsystem("transfer",
+            new WPI_VictorSPX(RobotMap.CAN.TRANSFER_VICTOR_1), new WPI_VictorSPX(RobotMap.CAN.TRANSFER_VICTOR_2));
+
+    /*
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: you don't do javadoc inside a method.

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+     * Places a command which moves the transfer subsystem on the SmartDashboard.
+     */
+    robotNamespace.putData("move transfer", new MoveGenericSubsystem(transfer, TRANSFER_SPEED));
+  }
+
+  @Override
+  public void robotPeriodic() {
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: add the periodic of the `robotNamespace`

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  /**
+   * A namespace is an object that holds values in a {@link NetworkTable}.
+   */
+  private RootNamespace robotNamespace;
+
+  /*
+   * A subsystem which transfers cargos. It consists of two MotorControllers that move a timing strap. The strap
+   * puts pressure on a cargo, and thus the cargo moves along with the strap.
+   */
+  MotoredGenericSubsystem transfer;
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: default visibility modifier?

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/transfer/Robot.java (hunk 2 above)
Comment (OfriRosenbaum) [reply src/main/java/frc/robot/subsystems/transfer/Robot.java]: also add the periodic of `transfer`

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+  @Override
+  public void robotInit() {
+    robotNamespace = new RootNamespace("robot");
+
+    transfer = new MotoredGenericSubsystem("transfer",
+            new WPI_VictorSPX(RobotMap.CAN.TRANSFER_VICTOR_1), new WPI_VictorSPX(RobotMap.CAN.TRANSFER_VICTOR_2));
+
+    /*
+     * Places a command which moves the transfer subsystem on the SmartDashboard.
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: change SmartDashboard to `{@link Shuffleboard}`

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+public class Robot extends TimedRobot {
+
+    public static final double TRANSFER_SPEED = 0.5;
+
+    /**
+     * A namespace is an object that holds values in a {@link NetworkTable}.
+     */
+    private RootNamespace robotNamespace;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: initialize the namespace here

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+    /*
+     * A subsystem which transfers cargos. It consists of two MotorControllers that move a timing strap. The strap
+     * puts pressure on a cargo, and thus the cargo moves along with the strap.
```
Comment (tal-sitton) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: puts pressure on a `ball`, 
`cargo` is too specific

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/transfer/Robot.java [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    public static final double TRANSFER_SPEED = 0.5;
+
+    /**
+     * A namespace is an object that holds values in a {@link NetworkTable}.
+     */
+    private final RootNamespace robotNamespace = new RootNamespace("robot");
+
+    /*
+     * A subsystem which transfers cargos. It consists of two MotorControllers that move a timing strap. The strap
+     * puts pressure on a ball, and thus the cargo moves along with the strap.
```
Comment (tal-sitton) [inline src/main/java/frc/robot/subsystems/transfer/Robot.java]: again, `cargo`?

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Pull request merged**